android/app/src/main/java/updated/mysterium/vpn/ui/create/account/CreateAccountActivity.kt

```diff
@@ -119,7 +119,7 @@ class CreateAccountActivity : BaseActivity() {
             br.close()
         } catch (exception: FileNotFoundException) {
```

Comment:
If we have same handle logic for FileNotFoundException and Exception, maybe we can have only Exception?

Reply:
Done